app/helpers/advanced_messenger_helper.rb

```diff
@@ -47,4 +47,34 @@ def getUnreadNotificationsForCurrentUserGroupByIssues()
     def getUnreadNotificationsForCurrentUserCount()
```

Comment:
Is this function called one single time? Or is it for each note? If it is for each note, cand you try and call it a single time?

Reply:
It is called only once per page, and it is called again when a notes gets read